src/Aspire.Hosting/Dcp/Process/ProcessUtil.cs

```diff
@@ -95,7 +95,7 @@ public static (Task<ProcessResult>, IAsyncDisposable) Run(ProcessSpec processSpe
 
         try
         {
-            AspireEventSource.Instance.ProcessLaunchStart(processSpec.ExecutablePath, processSpec.Arguments ?? "");
+            // AspireEventSource.Instance.ProcessLaunchStart(processSpec.ExecutablePath, processSpec.Arguments ?? "");
```

Comment:
Why?

Reply:
Shared code, I didn’t attempt to pull it over into azure provisioning